pkg/clients/clients.go

```diff
@@ -1,14 +1,17 @@
 package clients
 
 import (
-	"github.com/coreos/etcd/client"
-	"k8s.io/client-go/rest"
-	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+	etcd "github.com/coreos/etcd/client"
+	k8s "k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+	"sync"
```

Comment:
`goimports` would actually put "sync" at the top because it's from the stdlib. 
```
import (
    "sync"
    
    etcd "github.com/coreos/etcd/client"
    k8s "k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
)
```